Makefile

```diff
@@ -14,3 +14,10 @@ install-go:
 
 build-nsqd:
 	go/bin/go build -o bin/imunify-nsqd ./apps/nsqd/...
+
+github-release: TARGET ?= nsq-1.2.1-1.linux-amd64.go1.19.1
+github-release:
+	mkdir -p ${TARGET}
+	go build -o ${TARGET}/bin/nsqd ./apps/nsqd/...
+	go build -o ${TARGET}/bin/nsqlookupd ./apps/nsqlookupd/...
```

Comment:
1. I suggest to deliver only nsqd what we need (moreover, in case nslookupd we can deliver it separately in one release)
2. we don't need specific github-release, we just can "make build" and then use nsqd